docker-app/qfieldcloud/subscription/models.py

```diff
@@ -588,25 +592,15 @@ def active_users(self):
                 self.current_period_until,
             )
 
-        assert (
-            self.current_period_since == self.current_period_until
-        ), "Both current_period _since and _until must be set."
-
-        now = timezone.now()
-        month_ago = now - timedelta(days=28)
-
-        return self.account.user.active_users(
-            month_ago,
-            now,
-        )
```

Comment:
Fix on the way, not directly related to feature.

Reply:
ok, thanks for the clean-up